### PR TITLE
add application insights SDK

### PR DESCRIPTION
**Pull request by RaphHaddad**
@aaronpowell why are we checking in the packages/ folder?

### CODE REVIEW

**Comment by aaronpowell** [comment]
Err... because it was like that when I forked it 😛

File: DevDevDev/ErrorHandler/AiHandleErrorAttribute.cs [hunk 1]
```diff
@@ -0,0 +1 @@
...
                if (filterContext.HttpContext.IsCustomErrorEnabled)
                {   
                    var ai = new TelemetryClient();
                    ai.TrackException(filterContext.Exception);
                } 
            }
            base.OnException(filterContext);
        }
    }
}
```
Comment (aaronpowell) [inline DevDevDev/ErrorHandler/AiHandleErrorAttribute.cs]: Does this have a line ending issue?

File: DevDevDev/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -21,6 +21,18 @@
     <script src="@Url.Content("~/scripts/vendor/custom.modernizr.js")"></script>
     <script src="@Url.Content("~/scripts/vendor/jquery.js")"></script>
 
+    <script type = 'text/javascript' >
```
Comment (aaronpowell) [inline DevDevDev/Views/Shared/_Layout.cshtml]: Can you set the key in the `web.config` and inject it that way?

Comment (RaphHaddad) [inline DevDevDev/Views/Shared/_Layout.cshtml]: yes will do and commit&&push

**Comment by RaphHaddad** [comment]
Check if the build works. If it does i will remove the packages from git and put them on .gitignore.

File: DevDevDev/ErrorHandler/AiHandleErrorAttribute.cs (hunk 1 above)
Comment (RaphHaddad) [inline DevDevDev/ErrorHandler/AiHandleErrorAttribute.cs]: This was generated when I imported the SDK. Will see if i can standardise line endings.

File: DevDevDev/Web.config [hunk 3]
```diff
@@ -1,83 +1,117 @@
...
-    <add key="CaptchaPass" value=""/>
-    <add key="CaptchaSalt" value=""/>
-    <add key="CaptchaChars" value=""/>
+    <add key="webpages:Version" value="2.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="PreserveLoginUrl" value="true" />
+    <add key="ClientValidationEnabled" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="AzureTableStorageConnectionString" value="UseDevelopmentStorage=true" />
+    <add key="ApplicationInsightsInstrumentationKey" value="e8ea2985-e30c-4d1e-826f-5472e6bf25bb" />
```
Comment (aaronpowell) [inline DevDevDev/Web.config]: Is that the prod key? I'd prefer it not be in a public git repo 😛

**Comment by aaronpowell** [comment]
Looks to go fine if you remove the `packages` folder. Feel free to add that here too

File: DevDevDev/Web.config (hunk 3 above)
Comment (RaphHaddad) [inline DevDevDev/Web.config]: it's going to be on the JavaScript (public) anyway.

**Comment by RaphHaddad** [comment]
OK don't merge until i remove packages/

File: DevDevDev/Web.config (hunk 3 above)
Comment (aaronpowell) [inline DevDevDev/Web.config]: Well... true.

Smart-arse ;)

**Comment by RaphHaddad** [comment]
@aaronpowell . It seems a simple removal of 'packages/' isn't as straightforward.

Some packages are in the folder but not registered on packages.config.

Can we get these changes in and I can do another PR for the removal of packages.

**Pull request merged**